app/src/main/java/com/climbz/strongcode/FlexibilityActivity.java

```diff
@@ -0,0 +1,54 @@
+package com.climbz.strongcode;
+
+import android.os.Bundle;
+import android.support.design.widget.FloatingActionButton;
+import android.support.design.widget.Snackbar;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ListView;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FlexibilityActivity extends AppCompatActivity {
```

Comment:
this looks very similar to ConditioningActivity, should probs be combined into one activity